src/PhpSpreadsheet/Reader/Xlsx.php

```diff
@@ -2222,4 +2228,48 @@ private static function extractPalette(?SimpleXMLElement $sxml): array
 
         return $array;
     }
+
+    private function processIgnoredErrors(SimpleXMLElement $xml, Worksheet $sheet): void
+    {
+        $attributes = self::getAttributes($xml);
+        $sqref = (string) ($attributes['sqref'] ?? '');
+        $numberStoredAsText = (string) ($attributes['numberStoredAsText'] ?? '');
+        $formula = (string) ($attributes['formula'] ?? '');
+        $twoDigitTextYear = (string) ($attributes['twoDigitTextYear'] ?? '');
+        $evalError = (string) ($attributes['evalError'] ?? '');
+        if (!empty($sqref)) {
+            $explodedSqref = explode(' ', $sqref);
+            $pattern1 = '/^([A-Z]{1,3})([0-9]{1,7})(:([A-Z]{1,3})([0-9]{1,7}))?$/';
```

Comment:
Can we use any of the existing pattern constants?

Reply:
The patterns in Calculation are too complicated - they allow for worksheet names and dollar signs and case-insensitivity, none of which can occur here (and probably other stuff - I haven't analyzed the entire pattern). Are other more simple patterns defined elsewhere?